test/report-to-row.test.ts

```diff
@@ -1,13 +1,16 @@
-const path = require('path');
-const fs = require('fs');
-const { reportToRow, reportToRowHeaders } = require('../reportToRow');
+import * as path from 'path';
+import * as fs from 'fs';
+import { reportToRow, reportToRowHeaders } from '../report-to-row';
+
 const testCsvPath = path.join(__dirname, 'support', 'lombard.csv');
 const fileContents = fs.readFileSync(testCsvPath, { encoding: 'utf-8' });
 
 describe('reportToRow', () => {
   it('converts rows to columns', () => {
     const row = reportToRow(fileContents);
 
+    if (!Array.isArray(row)) throw new Error('expected an array');
```

Comment:
I added this here so that TS can know that the type of `row` is `CSVReportRow` instead of `CSVReportRow | false`. While Jest has its own way of checking if something is an array, Jest's types don't currently don't tell TS to narrow the type of `row` to be an array.